usaspending_api/submissions/management/commands/load_dabs_submission_window_schedule.py

```diff
@@ -0,0 +1,112 @@
+import logging
+
+from django.core.management.base import BaseCommand
+from django.db import connections, transaction
+
+from usaspending_api.etl.broker_etl_helpers import dictfetchall
+from usaspending_api.submissions.models import DABSSubmissionWindowSchedule
+
+logger = logging.getLogger("console")
+
+# SQL to create Month Period Schedules using broker table
+# Use all periods after Period 9, Year 2020 from table
+# Submission Due Date comes from 'publish_deadline' column
+MONTH_SCHEDULE_SQL = """
+select
+    make_date(year, period, 1) - interval '3 months' as period_start_date,
+    make_date(year, period, 1) - interval '2 months' - interval '1 day' as period_end_date,
+    period_start as submission_start_date,
+    certification_deadline as certification_due_date,
+    publish_deadline as submission_due_date,
+    publish_deadline + interval '1 day' as submission_reveal_date,
+    year as submission_fiscal_year,
+    (period + 2) / 3 as submission_fiscal_quarter,
+    period as submission_fiscal_month,
+    false as is_quarter
+from
+    submission_window_schedule
+where
+    period_start >= '2020-07-17'
+"""
```

Comment:
There are periods, but the logic is that you can't create a period 1 submission, and need to include period 1's data in with period 2. Since Broker hasn't put in FY2021 dates yet, it's hard to say what it will look like. This also comes down to:
- are we talking about submission periods (i.e. fiscal periods for which submissions can be made)
- or fiscal periods (in which case there _is_ a period 1, just not used for much)

We'll need to decide with POs.

Reply:
Unless I misinterpreted, I believe the "there is no month/period 1 and month/period 2 begin and end dates span two months" is verbatim what I was told by a Broker representative.  We (N. Royal We) definitely need to follow up.

Reply:
Yep - there is no period 1. 

So if you can adjust as Kirk has said, that period 2 actually spans 2 months (a bit tricky perhaps to do, or at least, doesn't follow the pattern. Could start at 3 and let period 2 be a 1-off injection)

Reply:
Will this be reflected in the broker table? As long as period 2 has the appropriate period_start and publish_deadline columns, there is no need to hard code anything.

Reply:
This was confirmed in the Slack conversation: https://data-transparency-bfs.slack.com/archives/C0122HUCD17/p1593103387330000?thread_ts=1593100496.319600&cid=C0122HUCD17

> Does period_start to publish_deadline in the submission_window_schedule table for period 2 represent the submission window for period 2? - In which submissions for both period 1 and period 2 data will be made?
Yes and yes.

Reply:
I added a condition to preventing period 1 from being used to create monthly schedules.

Reply:
My concern was more about `period_start_date`... is that supposed to be `YYYY-10-01` or `YYYY-11-01` for period 2?  "There is no period 1" is less helpful than it seems like it should be.  What do we use it for?  It gets copied into File A/B/C tables and is used for filtering.  Will we ever have a need to filter by `YYYY-10-01`?  I have no idea.

You know what, if nobody knows the answer, let's just push what we have and fix it if we're wrong.